integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/message/processor/test/DeactivatedCappMPBehaviourOnRestartTestCase.java

```diff
@@ -80,15 +68,13 @@ public void testDeactivatedMPUponServerRestart() throws Exception {
         axis2Server.start();
 
         // Restart the ESB Server after the MP is deactivated.
-        serverConfigurationManager.restartGracefully();
+        CarbonServerExtension.restartServer();
 
         super.init();
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
+        CarbonLogReader carbonLogReader = new CarbonLogReader();
 
-        Assert.assertFalse(Utils.checkForLog(logViewerClient, EXPECTED_ERROR_MESSAGE, 10),
+        Assert.assertFalse(Utils.logExists(carbonLogReader, EXPECTED_ERROR_MESSAGE, 10),
                 "Error occurred while deploying Carbon Application!");
-        Assert.assertTrue(Utils.checkForLog(logViewerClient, SUCCESS_MESSAGE, 10),
```

Comment:
We need this assert too to see whether Mp deployed successfully.

Reply:
I removed it since we're removing the `uploadCapp()` call [here](https://github.com/wso2/micro-integrator/pull/336/files#diff-a1a50aa3c0dc71fac52b25f6529d8a38L62)

Reply:
Fixed.